enterprise/com/src/main/java/org/neo4j/com/storecopy/TransactionCommittingResponseUnpacker.java

```diff
@@ -230,6 +231,10 @@ public void obligation( long txId ) throws IOException
             {
                 obligationFulfiller.fulfill( txId );
             }
+            catch ( IllegalStateException e )
+            {
+                throw new ComException( "Failed to pull updates", e );
+            }
             catch ( InterruptedException e )
             {
                 throw new IOException( e );
```

Comment:
But this state would not be caused by an interrupt I think. Or do you see any situation that might occur?